lib/ReportFunctions.groovy

```diff
@@ -0,0 +1,160 @@
+
+
+
+
+class ReportFunctions{
```

Comment:
Could you maybe run the VSCode formatter on this file to clean up whitespace, etc?